WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsPresentingViewController.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+import UIKit
+
+final class CardReaderSettingsPresentingViewController: UIViewController {
+
+    /// An array of viewModels and related view classes
+    private var viewModelsAndViews = [CardReaderSettingsViewModelAndView]()
+
+    /// Set our dependencies
+    func configure(viewModelsAndViews: [CardReaderSettingsViewModelAndView]) {
```

Comment:
I am not sure if this makes actual sense, but if if were possible to inject this via the constructor, we could get away with declaring viewModelsAndViews as a let instead of a var

Reply:
Whoops, storyboards. Scratch my comment out please 😊